R/create_query_data.R

```diff
@@ -258,18 +211,18 @@ create_query_data <- function(queries,
   query_data <- vector("list", length(queries))
   for (i in seq_along(queries)) {
     # get term names and term variable
-    if (inherits(queries[[i]]$definition, "smq_select")) {
-      # query is a SMQ
+    if (inherits(queries[[i]]$definition, "basket_select")) {
+      #query is a basket
       query_data[[i]] <- get_terms_from_db(
-        version = meddra_version,
-        fun = get_smq_fun,
+        version = version,
+        fun = get_terms_fun,
         queries = queries,
         definition = queries[[i]]$definition,
         expect_query_name = TRUE,
         expect_query_id = !is.null(queries[[i]]$id),
         i = i,
         temp_env = temp_env,
-        type = "SMQ"
+        type = queries[[i]]$definition$type
```

Comment:
This line should be removed as the `type` argument was removed from `get_terms_from_db()`.